tests/bud.bats

```diff
@@ -390,6 +390,23 @@ _EOF
   assert "$output" !~ "unwanted stage"
 }
 
+# Note: Please skip this tests in case of podman-remote build
+@test "build: test race in updating image name while performing parallel commits" {
+  # Run 25 parallel builds using the same Containerfile
+  local count=25
+  for i in $(seq --format '%02g' 1 $count); do
+      timeout --foreground -v --kill=10 60 \
+              ${BUILDAH_BINARY} ${BUILDAH_REGISTRY_OPTS} ${ROOTDIR_OPTS} $WITH_POLICY_JSON build --squash --iidfile ${TEST_SCRATCH_DIR}/id.$i --timestamp 0 -f $BUDFILES/check-race/Containerfile &>/dev/null &
```

Comment:
Also, you may want to change the above to `... --quiet ... >/dev/null` (add `--quiet`, remove `&` so stderr is preserved).